src/screens/benefit/Details.tsx

```
@@ -1,408 +1,435 @@
+	previous_year_marks?: string;
+	phone_number?: string;
+	username: string;
+	email: string;
 }
 
 interface WebFormProps {
-  url?: string;
-  formData?: {};
-  item?: BenefitItem;
+	url?: string;
+	formData?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)